package.json

```diff
@@ -54,6 +54,9 @@
     "test": "NODE_ENV=test jest"
   },
   "jest": {
+    "globals": {
+      "__DEV__": true
```

Comment:
yeah I saw and removed my comment. Didn't know we had this config, this kinda sucks because the config is not very useful. The env setup script exposes both `global` and `window` to which you can attach globals.